rust/core-lib/src/view.rs

```diff
@@ -612,9 +601,7 @@ impl View {
                     } else {
                         let start_line = seg.our_line_num;
                         let end_line = start_line + seg.n;
-                        if self.hls_dirty {
-                            self.update_find_for_lines(text, start_line, end_line);
-                        }
```

Comment:
I don't totally understand the logic here.. is this related to how we now find all instances, instead of just for the active region?